lib/saml/url_service.rb

```diff
@@ -53,9 +53,12 @@ def base_redirect_url
 
     # rubocop:disable Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
     def login_redirect_url(auth: 'success', code: nil)
-      return verify_url if auth == 'success' && user.loa[:current] < user.loa[:highest]
+      if auth == 'success'
+        return verify_url if user.loa[:current] < user.loa[:highest]
 
-      return @tracker&.payload_attr(:redirect) if @tracker&.payload_attr(:redirect)
```

Comment:
for the cerner redirect, if the user authentication fails, do we want to show them the va.gov error page or redirect them back to cerner for them to show an error?

Reply:
I...don't know. Question for @bridgethapner if we've specified that

Reply:
My concern would be that Cerner would try to send them back to the sign-in page again and they'd be stuck in a loop with no real feedback

Reply:
I think we should deploy this as-is, wherein it will show an error message, while we iterate with Cerner et al about whether we should redirect them back to any particular destination.

Reply:
I'm ok with that, I just wanted to make sure it's intentional to redirect cerner users to the va.gov failure page.